enigma-server/docs/protocol.md

```diff
@@ -278,9 +286,10 @@ struct SyncMappingsS2CPacket {
 }
 struct MappingNode {
     NoParentEntry obf_entry;
-    boolean is_named;
     utf name;
     utf javadoc;
+    unsigned short token_type;
```

Comment:
probably worth auditing the token type on the server. TODO add a readTokenType in PacketHelper that validates this